common/src/Model/Brush.cpp

```diff
@@ -1184,8 +1184,10 @@ namespace TrenchBroom {
             brushes.reserve(result.size());
 
             for (const auto& geometry : result) {
-                auto* brush = createBrush(factory, worldBounds, defaultTextureName, geometry, subtrahends);
-                brushes.push_back(brush);
+                try {
+                    auto* brush = createBrush(factory, worldBounds, defaultTextureName, geometry, subtrahends);
```

Comment:
Not necessary right now, but `createBrush`, the Brush constructor, and ModelFactory::createBrush need documentation that they throw GeometryException.

Exceptions are such a pain because of how they propagate invisibly across API's... just while reviewing this, I noticed `VertexToolBase::csgConvexMerge` has no try block. Since that is called directly in `MapFrame::OnEditCsgConvexMerge` with no try block, we'll probably crash if a CSG merge ever fails.

Reply:
Yeah I wish C++ had checked exceptions like Java.

Reply:
I copied this concern about csg merge to https://github.com/kduske/TrenchBroom/issues/2744